atom/browser/api/atom_api_web_contents.cc

```diff
@@ -548,6 +559,58 @@ void WebContents::SetAllowTransparency(bool allow) {
   }
 }
 
+content::ServiceWorkerContext* WebContents::GetServiceWorkerContext() {
```

Comment:
It is preferred to put utility functions in anonymous namespace instead of making them member function.